modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.scss

```diff
@@ -37,6 +37,16 @@
 			}
 		}
 
+		.modal-content {
+			--modal-bottom-spacing: 120px;
+
+			.modal-body {
+				max-height: calc(100vh - var(--modal-bottom-spacing));
+				overflow-y: auto;
+			}
+		}
+
+
```

Comment:
Please remove this empty line

Reply:
You are in the wrong organization... I see Detra instead of Liferay

Reply:
@luca-pellizzon, I had already given my GitHub id on my first day, but anyone added me to the organization. I'll send a request, following the instructions @smottal told me in a Slack chat. 
PS: it's possible to have more than one organization on GitHub :)